README.rst

```diff
@@ -64,15 +64,15 @@ PyInstaller is available on PyPI. You can install it through `pip`::
 Requirements and Tested Platforms
 ------------------------------------
 
-- Python: 
+- Python:
```

Comment:
No whitespace changes please - that includes all of the other changes you made. The tabs are there for a reason!

Reply:
I'll revert them all (... but why? They seemed quite inconsistent around the file, that's why I removed them.)

Reply:
@naufraghi if you wish a detailed explanation ask htgoebel. He's the one who insists on it - and as the project head...

Reply:
@Legorooj @naufraghi The reason is explained in the manual https://pyinstaller.readthedocs.io/en/stable/development/commit-messages.html#in-detail